tests/ssl-opt.sh

```diff
@@ -1583,6 +1583,23 @@ run_test    "Opaque key for server authentication" \
             -S "error" \
             -C "error"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
+requires_config_enabled MBEDTLS_X509_CRT_PARSE_C
+requires_config_enabled MBEDTLS_ECDSA_C
+requires_config_enabled MBEDTLS_SHA256_C
+run_test    "Opaque key for server authentication (ECDH-)" \
+            "$P_SRV auth_mode=required key_opaque=1 crt_file=data_files/server5.ku-ka.crt \
```

Comment:
Please also force the version to `tls1_2` to avoid trouble when we add server-side support for TLS 1.3.

Also, nice way of forcing the use of a static ECDH ciphersuite - I had `force_ciphersuite` in mind, but it's not as good at it forces more things (cipher etc) so causes more dependencies, hence skipping the test in more configurations, so I like the way you did it better.